qiita_db/processing_job.py

```diff
@@ -429,12 +430,49 @@ def release_validators(self):
                         JOIN qiita.processing_job_status USING
                             (processing_job_status_id)
                      WHERE pjv.processing_job_id = %s
-                        AND processing_job_status != %s"""
-            qdb.sql_connection.TRN.add(sql, [self.id, 'waiting'])
+                        AND processing_job_status NOT IN %s"""
+            sql_args = [self.id, ('waiting', 'error')]
```

Comment:
based on the description it sounds like here it should only be waiting, right? If this is incorrect, what happens with erred jobs?